src/communication/azext_communication/aaz/latest/communication/_create.py

```diff
@@ -0,0 +1,389 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "communication create",
+)
+class Create(AAZCommand):
+    """Create a new CommunicationService or update an existing CommunicationService.
+
+    :example: Create a communication resource
+        az communication create -n <your-resource-name> -g <your-resource-group>
```

Comment:
```suggestion
        az communication create -n ResourceName -g ResourceGroupName
```

Reply:
BTW, the create command has some many arguments, you should provide a more useful example.